Bento/Helpers.swift

```diff
@@ -63,3 +63,11 @@ extension Optional {
         return selector(unwrapped, other)
     }
 }
+
+internal func search<T>(from leaf: UIView, type: T.Type) -> T? {
+    var cell = leaf.superview
+    while let view = cell, !(view is T) {
+        cell = view.superview
+    }
+    return cell as! T?
```

Comment:
Why not `cell as? T`?

Reply:
This is an explicit assertion. It must be `T` or `nil` at this point.